package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Seneca RabbitMQ transport",
   "main": "rabbitmq-transport.js",
   "scripts": {
-    "test": "node_modules/.bin/mocha test/*.test.js"
+    "test": "./node_modules/.bin/lab test/*test.js  -r console -v",
+    "setup-test": "./setup.sh"
```

Comment:
Ah I misunderstood, in this case leave it for the moment. I want to talk to @geek about standard names for all our commands.